operator/internal/central/reconciler/reconciler.go

```diff
@@ -34,8 +34,18 @@ func RegisterNewReconciler(mgr ctrl.Manager, selector string) error {
 		pkgReconciler.WithPreExtension(extensions.ReconcileScannerDBPasswordExtension(mgr.GetClient(), mgr.GetAPIReader())),
 		pkgReconciler.WithPreExtension(extensions.ReconcileScannerV4DBPasswordExtension(mgr.GetClient(), mgr.GetAPIReader())),
 		pkgReconciler.WithPreExtension(extensions.ReconcileAdminPasswordExtension(mgr.GetClient(), mgr.GetAPIReader())),
-		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentral, extensions.DefaultCentralPVCName)),
-		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentralDB, extensions.DefaultCentralDBPVCName)),
+		pkgReconciler.WithPreExtension(extensions.ReconcilePVCExtension(mgr.GetClient(), mgr.GetAPIReader(), extensions.PVCTargetCentral, extensions.DefaultPVCValues{
+			ClaimName: extensions.DefaultCentralPVCName,
+			Size:      extensions.DefaultPVCSize,
```

Comment:
This is a bit verbose... how about keeping the claim name a required arg as it was, and using the [option pattern](https://dev.to/kittipat1413/understanding-the-options-pattern-in-go-390c) to optionally override the size from the default one? 🤔

Reply:
I don't think a notion of a default claim name makes sense, while a default size does, so I'd suggest an option only for the latter.

Reply:
After trying to implement something along those lines I'm having a second though. To make something like `WithClaimSize`  there has to be a structure in memory, which we're going to mutate with the new value. Since the whole thing returns a set of wrappers, the only reasonable candidate for such structure is `reconcilePVCExtensionRun` -- but it's private to the extensions package, and I'm afraid making it public will come at costs of less isolation. With that in mind I'm inclined to think the `DefaultPVCValues` structure sounds better, unless you've got some other way of implementing the "Options" pattern?

Reply:
I don't think you need to export the `reconcilePVCExtensionRun` type to make it work? 🤔 
For example:
```
type Option func(*priv)

type priv struct {
	foo string
}

func WithFoo(foo string) Option {
	return func(p *priv) {
		p.foo = foo
	}
}

func New(opts ...Option) string {
	p := &priv{}
	for _, o := range opts {
		o(p)
	}
	return p.foo
}
```

Works fine from another package:
```
	fmt.Printf("Hello and welcome, %s!\n", pkg2.New(pkg2.WithFoo("world")))
```

Reply:
To be completely honest, I don't like this version that much either. It looks that it separates logic in a way that subjectively feels bizarre to me :) Maybe it actually makes sense to not expose any defaults outside of the extensions package. But as long as if you folks are ok with that, I'm fine -- here is the WithOption version. Any commentaries?